lib/worker/fake-tty.js

```diff
@@ -2,19 +2,30 @@
 const tty = require('tty');
 const options = require('./options').get();
 
-if (options.tty) {
-	Object.assign(process.stdout, {isTTY: true}, options.tty);
+const fakeTTYs = [];
 
-	if (options.tty.colorDepth !== undefined) {
-		process.stdout.getColorDepth = () => options.tty.colorDepth;
+const isatty = tty.isatty;
+tty.isatty = fd => {
+	if (fakeTTYs.indexOf(fd) !== -1) {
```

Comment:
`fakeTTYs` may be better as a `Set` rather than an array. You could then simplify this function to `fd => fakeTTYs.has(fd) || isatty(fd)`.